oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/IndexRequest.java

```diff
@@ -29,5 +31,10 @@ public final class IndexRequest {
     private final String index;
     private final String type;
     private final String id;
+    /**
+     * the routing value to control the sharding policy
+     */
+    @Builder.Default
+    private final Optional<String> routing = Optional.empty();
```

Comment:
```suggestion
    /**
     * The routing value of the request.
     */
    @Builder.Default
    private final Optional<String> routing;
```

I think with @Builder.Default, this should have value as always.
https://stackoverflow.com/questions/35739670/what-is-the-default-for-optional-data-members-in-lombok-using-builder

Reply:
https://projectlombok.org/features/Builder
> If a certain field/parameter is never set during a build session, then it always gets 0 / null / false. If you've put @Builder on a class (and not a method or constructor) you can instead specify the default directly on the field, and annotate the field with @Builder.Default:
@Builder.Default private final long created = System.currentTimeMillis();

as official document, both operations require:
1. specify the default directly on the field
2. and annotate the field with @Builder.Default:

Reply:
OK, then please update the comments only.